cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -2631,6 +2631,15 @@
   </property>
 
   <property>
+    <name>security.keytab.path</name>
+    <value></value>
+    <description>
+      Configuration for specifying the location of kerberos keytabs. The location can contain ${name}
```

Comment:
maybe make it more clear these keytabs are for impersonation?  I can imagine somebody putting the path of ``cdap.master.kerberos.keytab`` here